docs/usage/python.md

```diff
@@ -88,6 +89,6 @@ In that case this would enable _only_ `npm`:
 
 ## Future work
 
-Feature requests are open for conda support and additional file types (e.g. `setup.cfg`).
+Feature requests are open for Conda support and additional file types (e.g. `setup.cfg`).
 You can locate these issues by filtering on the [#python](https://github.com/renovatebot/renovate/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%23python) hashtag in the repository.
 Please +1 and/or add a comment to each issue that would benefit you so that we can gauge the popularity/importance of each.
```

Comment:
>You can locate these issues by filtering on the [#python](https://github.com/renovatebot/renovate/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%23python) hashtag in the repository.
> Please +1 and/or add a comment to each issue that would benefit you so that we can gauge the popularity/importance of each.

This section seems out of place, do we still need this?

Reply:
Let's remove this "Future work" section completely

Reply:
And it's gone. 😄